test/amqp-integration-test.py

```diff
@@ -53,7 +54,8 @@ def verify_ocsp_good(certFile, url):
     output = get_ocsp(certFile, url)
     if not re.search(": good", output):
         print "Expected OCSP response 'good', got something else."
-        die(ExitStatus.OCSPFailure)
+        return False
+    return True
```

Comment:
This should specifically look for `unauthorized` in the response (I think), and return False only if it gets that instead of `: good`. If neither matches, the test should still die.